native/storybook/rtl-addon/register.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import addons from '@storybook/addons';
+
+const styles = {
+  notesPanel: {
+    margin: 10,
+    fontFamily: 'Arial',
+    fontSize: 14,
+    color: '#444',
+    width: '100%',
+    overflow: 'auto',
+  },
+};
+
+class Panel extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      rtlEnabled: false,
+    };
+  }
+  toggleRtl = () => {
+    this.setState({
+      rtlEnabled: !this.state.rtlEnabled,
+    });
+    const channel = addons.getChannel();
+    channel.emit('rtl-toggle', this.state.rtlEnabled);
```

Comment:
I think we should make `rtl-toggle` a constant in a `constants.js` file, so that it can be read both here and in `index.js`.